src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import (
+    Release, ReleaseCommit, Commit, CommitFileChange, Event
+)
+
+from itertools import izip
+from collections import defaultdict
+
+
+class EventFileCommittersEndpoint(ProjectEndpoint):
+
+    def _get_frame_paths(self, event):
```

Comment:
@mitsuhiko is there a better/more reliable way to get this information? This seems like a pattern we'd have repeated.